config/environments/development.rb

```diff
@@ -4,12 +4,12 @@
 Argo::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb
 
-  config.assets.compress = false
+  config.assets.compress = Settings.APPLICATION.ASSETS.COMPRESS
```

Comment:
at present, it's a setting that differs between default laptop configurations and argo-dev's deployed configuration, though both instances go by same the rails env name ("development").  i guess we could make it false in both setups?  though i think the difference is actually reasonable (there's presumably a slight performance gain, and editing code in place on argo-dev is rare).  or, we could differentiate laptop configs from the deployed "development" environment by using a different env name, though they are largely similar.  and adding another environment config file like that if it isn't really needed feels a little worse (to me) than adding new settings entries.

i don't know... this is a basic rails setting, so it seems like an appropriate thing to do, but i'm fine switching approaches.  i think my next choice, for simplicity's sake, would be to just make it false in all cases for this env and get rid of the setting.  but i vote for leaving it as it is in this branch, unless i'm missing something.